my_spot:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50  
  
  # Position Controllers ---------------------------------------
  joint1_position_controller_left_front:
    type: effort_controllers/JointPositionController
    joint: joint1_left_front
    pid: {p: 100.0, i: 0.01, d: 10.0}
  joint2_position_controller_left_front:
    type: effort_controllers/JointPositionController
    joint: joint2_left_front
    pid: {p: 100.0, i: 0.01, d: 10.0}
  joint3_position_controller_left_front:
    type: effort_controllers/JointPositionController
    joint: joint3_left_front
    pid: {p: 100.0, i: 0.01, d: 10.0}


  joint1_position_controller_left_rear:
    type: effort_controllers/JointPositionController
    joint: joint1_left_rear
    pid: {p: 100.0, i: 0.01, d: 10.0}
  joint2_position_controller_left_rear:
    type: effort_controllers/JointPositionController
    joint: joint2_left_rear
    pid: {p: 100.0, i: 0.01, d: 10.0}
  joint3_position_controller_left_rear:
    type: effort_controllers/JointPositionController
    joint: joint3_left_rear
    pid: {p: 100.0, i: 0.01, d: 10.0}



  joint1_position_controller_right_front:
    type: effort_controllers/JointPositionController
    joint: joint1_right_front
    pid: {p: 100.0, i: 0.01, d: 10.0}
  joint2_position_controller_right_front:
    type: effort_controllers/JointPositionController
    joint: joint2_right_front
    pid: {p: 100.0, i: 0.01, d: 10.0}
  joint3_position_controller_right_front:
    type: effort_controllers/JointPositionController
    joint: joint3_right_front
    pid: {p: 100.0, i: 0.01, d: 10.0}


  joint1_position_controller_right_rear:
    type: effort_controllers/JointPositionController
    joint: joint1_right_rear
    pid: {p: 100.0, i: 0.01, d: 10.0}
  joint2_position_controller_right_rear:
    type: effort_controllers/JointPositionController
    joint: joint2_right_rear
    pid: {p: 100.0, i: 0.01, d: 10.0}
  joint3_position_controller_right_rear:
    type: effort_controllers/JointPositionController
    joint: joint3_right_rear
    pid: {p: 100.0, i: 0.01, d: 10.0}


